secure-storage-sample/watch.sh

```diff
@@ -0,0 +1,5 @@
+echo "Loading plugin..."
+uxp plugin load
+
+echo "Watching plugin..."
+nodemon --exec "uxp plugin reload" -e js,jsx,html

```

Comment:
Let's remove this file (no fault here, but we don't need to distribute this file.)